tests/playwright/src/model/pages/resource-card-page.ts

```diff
@@ -0,0 +1,40 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import { type Locator, type Page } from '@playwright/test';
+
+import { ResourcesPage } from './resources-page';
+
+export class ResourceCardPage extends ResourcesPage {
```

Comment:
Do not inherit from `ResourcePage`. Card class should not have access to the whole parent content as we are looking for specialization in this case. If you look at how `ExtensionsPage` and ExtensionsCardPage are implemented, you can use the same class members and similar logic/behavior.